CONTRIBUTING.md

```diff
@@ -58,7 +58,7 @@ The other option would be setting up https://github.com/tjdevries/tree-sitter-lu
 - Install Treesitter, either with package manager or with github release
 - Install plugin as usual
 - cd to plugin
-- `mkdir -p build parser` sadly does doesn't exist
+- `mkdir -p build parser` sadly those don't exist
```

Comment:
Shouldn't be here, can you rebase? The first commit should not be part of this PR

Reply:
Sorry, I don't understand. Do you mean that this was already fixed? In that case, we can close this.

Reply:
Nvm, I understood. I thought I had submitted a separate PR. I'll revert these changes.